seminars/practice_2_string.py

```diff
@@ -1,200 +1,229 @@
-# make_abba('What', 'Up') → 'WhatUpUpWhat'
-# make_abba('', 'y') → 'yy'
-
-
-# Task 5
-def reverse_word(sentence: str) -> str:
-    """
-    Write a function that takes in a string of one or more words,
-    and returns the same string, but with all five or more letter words reversed.
-
-    Strings passed in will consist of only letters and spaces.
-    Spaces will be included only when more than one word is present.
-    """
-    # student realisation goes here
-
-
-# reverse_word("Hey fellow warriors") == "Hey wollef sroirraw"
-#
-# reverse_word("This is a test") == "This is a test"
-#
-# reverse_word("This is another test") == "This is rehtona test"
-
-
-# Task 6
-def generate_hashtag(input_string: str) -> str:
-    """
-    The marketing team is spending way too much time typing in hashtags.
-    Let's help them with our own Hashtag Generator!
-
-    Here's the deal:
-
-    It must start with a hashtag (#).
-    All words must have their first letter capitalized.
-    If the final result is longer than 140 chars it must return false.
-    If the input or the result is an empty string it must return false.
-    Examples
-    " Hello there thanks for trying my quiz"  =>  "#HelloThereThanksForTryingMyQuiz"
-    "    Hello     World   "                  =>  "#HelloWorld"
-    ""                                        =>  false
-    """
-    # student realisation goes here
-
-
-# Task 7:
-def combo_string(first_string: str, second_string: str) -> str:
-    """
-    Given two strings, concatenate like the following: shorter+longer+shorter
-    """
-    # student realisation goes here
-
-
-# combo_string('Hello', 'hi') → 'hiHellohi'
-# combo_string('hi', 'Hello') → 'hiHellohi'
-# combo_string('aaa', 'b') → 'baaab'
-# combo_string('', 'bb') → 'bb'
-# combo_string('aaa', '1234') → 'aaa1234aaa'
-# combo_string('bb', 'a') → 'abba'
-
-
-# Task 1: advanced
-def string_splosion(input_string: str) -> str:
-    """
-    Given the string, format it like in the example.
-    """
-    # student realisation goes here
-
-
-# Function calls with expected result:
-# string_splosion('Code') → 'CCoCodCode'
-# string_splosion('abc') → 'aababc'
-# string_splosion('ab') → 'aab'
-# string_splosion('Kitten') → 'KKiKitKittKitteKitten'
-# string_splosion('x') → 'x'
-
-
-# Task 2: advanced
-def string_match(first_string: str, second_string: str) -> int:
-    """
-    Given two strings, find the number of times an arbitrary substring (with length of 2)
-    is found at the same position in both strings.
-    """
-    # student realisation goes here
-
-# Function calls with expected result:
-# string_match('xxcaazz', 'xxbaaz') → 3
-# NOTE: «xx», «aa» and «az» are found at the same position in both strings
-# string_match('abc', 'abc') → 2
-# string_match('abc', 'axc') → 0
-# string_match('he', 'hello') → 1
-# string_match('h', 'hello') → 0
-# string_match('aabbccdd', 'abbbxxd') → 1
-# string_match('aaxxaaxx', 'iaxxai') → 3
-# string_match('iaxxai', 'aaxxaaxx') → 3
+# """
```

Comment:
revert, please

Reply:
If you do not know how - ask in chat. Or just copy the content of the file from https://github.com/fipl-hse/2023-2-level-labs/blob/main/seminars/practice_2_string.py